suite/tests/common/nzcv.c

```diff
@@ -76,6 +76,14 @@ main()
     uint i;
     INIT();
 
+    /* We use this test binary with our libcallstack sample library to track malloc in
+     * sample.callstack test. So, we make sure we have at least one malloc call at the
```

Comment:
Does sample.callstack use both? I thought it used only common.eflags.

Reply:
It uses `common.eflags` in x86 and `common.nzcv` in aarch64, but the test name is the same `sample.callstack`.
Added the binary name though.